joblib/parallel.py

```diff
@@ -137,10 +137,7 @@ def __call__(self, *args, **kwargs):
             e_type, e_value, e_tb = sys.exc_info()
             text = format_exc(e_type, e_value, e_tb, context=10,
                               tb_offset=1)
-            if issubclass(e_type, TransportableException):
-                raise
```

Comment:
This was actually introduced recently by https://github.com/joblib/joblib/pull/207. In some sense this is a particular case of what this PR is trying to solve. The code is simpler without these two lines and the important information (i.e. worker traceback) is in `text` rather than in the main process traceback.